### PR TITLE
Feature/tm 30 assign to user

### PR DESCRIPTION
**Pull request by Rihards-B**

### CODE REVIEW

File: src/database/dummy-users.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+[
+    {
+        "_id": null,
+        "username": "Rihards-B",
+        "password": "password",
+        "first_name": "Rihards",
+        "last_name": "Blekus"
+    },
+    {
+        "_id": null,
```
Comment (wanpii) [inline src/database/dummy-users.json:10]: should id be null?

File: src/database/dummy-tasks.json [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
-        "id": "2",
+        "_id": null,
         "title": "TM-6 List Tasks",
         "description": "Display a list of all tasks",
         "type": "Task",
         "status": "complete"
     },
     {
-        "id": "3",
+        "_id": null,
```
Comment (wanpii) [inline src/database/dummy-tasks.json:17]: should id be null?

File: src/database/initialize.ts [hunk 3]
```diff
@@ -9,7 +11,11 @@ export const initialize = async () => {
         await TaskModel.deleteMany();
         console.log("Creating dummy tasks");
         await TaskModel.create(tasksJson);
-    } catch(error) {
+        console.log("Dropping users table!");
+        await UserModel.deleteMany();
+        console.log("Creating users");
+        await UserModel.create(usersJson);
```
Comment (wanpii) [inline src/database/initialize.ts:17]: I assume it generates the id here for users and tasks, is that why you set _id to null?

Comment (Rihards-B) [reply src/database/initialize.ts:17]: Yes, not passing an _id, generates a unique one

**Pull request merged**